src/nix/nix.md

```diff
@@ -130,6 +130,19 @@ the Nix store. Here are the recognised types of installables:
   If you want to operate on the store derivation itself, pass the
   `--derivation` flag.
 
+* **Indexed store derivations**: `/nix/store/p7gp6lxdg32h4ka1q398wd9r2zkbbz2v-hello-2.10.drv^out`
+
+  *(Experimental, part of by the `computed-derivations` experimental feature.)*
+
+  Store derivations can be indexed with a non-empty comma-separated list
+  of specific output names, or `*` meaning all ouptuts. This allows
+  finer control versus just specifying a derivation (without
+  `--derivation`) and getting all the outputs.
+
+  This is especially useful for (currently unstable) floating content
+  addressed derivations, which do not have precomputed output paths that
+  can be used instead.
+
```

Comment:
I think this section can be generalized, since the `^` syntax is supported for other types of installables (e.g. `nixpkgs#hello^bin`). We can start with this description from the 2.9 release notes:
```
* You can now specify which outputs of a derivation `nix` should
  operate on using the syntax `installable^outputs`,
  e.g. `nixpkgs#glibc^dev,static` or `nixpkgs#glibc^*`. By default,
  `nix` will use the outputs specified by the derivation's
  `meta.outputsToInstall` attribute if it exists, or all outputs
  otherwise.
```

Reply:
Indeed it can, in fact you already added even more complete docs below for that I will just fold the new stuff into.